README.md

```diff
@@ -1,12 +1,40 @@
 # Missed Issues
 
-A CLI tool for finding tickets that mention a team by no one has responded to from the team.
+A CLI tool for finding issues that mention a team and no one from the team has responded to the issue.
 
-`missed-issues --org mapbox --team teamname`
+`missed-issues --org mapbox --team teamname --token github_token`
 
 This will list all issues in the mapbox org that mention `@mapbox/teamname` where no member of that team has replied to the issue since the mention.
```

Comment:
Do we need to run this again for osrm-text-instructions, or do we not use team tags in public repos? And/or how should we monitor for activity in there?

Reply:
github teams are organization specific so you'd need to run this per github org you wanted reporting on.

This will work in public and private issues. Since this repo is public, any issue here that mentions @mapbox/guidance will show if someone in the mapbox org ran `missed-issues --team guidance --org mapbox`.